src/web/resources/report-page.js

```diff
@@ -0,0 +1,400 @@
+            detailsState = !detailsState
+        }
+    })
+
+    return Element("div", { id: "filters" }, [details])
+}
+
+function buildCheckboxLabel(idTag, text, boolState) {
+    return Element("label", { classList: idTag }, [
+        Element("input", { type: "checkbox", checked: boolState }, []),
+        text])
+}
+
+function updateChildren(children, state) {
+    children.forEach((child) => {
+        if (state) {
+            child.classList.remove("hidden")
+        } else {
+            child.classList.add("hidden")
+        }
+    })
+}
+
+function plotXY(testsData) {
+    var filteredTests = []
+    testsData.forEach((test) => {
+        if (filterState[test.status]) {
+            filteredTests.push(test)
+        }
+    })
+    const out = Plot.plot({
+        marks: [
+            Plot.line([[0, 0], [1, 1]], { stroke: '#ddd' }),
+            on(Plot.dot(filteredTests, {
+                x: d => 1 - d.start / 64, y: d => 1 - d.end / 64,
+                fill: "#00a", strokeWidth: 2,
+            }), {
+                click: (e, d) => { window.location = d.link + "/graph.html"; },
+            }),
+        ],
+        className: "clickable",
+        marginBottom: 0,
+        marginRight: 0,
+        width: '400',
+        height: '400',
+        x: { nice: true, line: true, tickFormat: "%", },
+        y: { nice: true, line: true, tickFormat: "%", },
+    })
+    out.setAttribute('viewBox', '0 0 420 420')
+    return out;
+}
+
+function plotPareto(jsonData) {
+    const [initial, frontier] = jsonData["merged-cost-accuracy"];
+    const out = Plot.plot({
+        marks: [
+            Plot.dot([initial], {
+                stroke: "#d00",
+                symbol: "square",
+                strokeWidth: 2
+            }),
+            Plot.line(frontier, {
+                stroke: "#00a",
+                strokeWidth: 2,
+            }),
+        ],
+        width: '400',
+        height: '400',
+        x: { line: true, nice: true, tickFormat: c => c + "×" },
+        y: { line: true, nice: true, domain: [0, 1], tickFormat: "%", },
+        marginBottom: 0,
+        marginRight: 0,
+    })
+    out.setAttribute('viewBox', '0 0 420 420')
+    return out;
+}
+
+function tableBody(jsonData) {
+    var rows = []
+    for (let test of jsonData.tests) {
+        if (filterState[test.status]) {
+            rows.push(tableRow(test, rows.length))
+        }
+    }
+    return Element("tbody", {}, rows)
+}
+
+function fixTarget(targetPer) {
+    // HACK be smarter
+    if (targetPer == "100.0%") {
+        return ""
+    } else {
+        return targetPer
+    }
+}
+
+function tableRow(test, i) {
+    const tr = Element("tr", { classList: test.status }, [
+        Element("td", {}, [test.name]),
+        Element("td", {}, [formatAccuracy(test.start, test.bits)]),
+        Element("td", {}, [formatAccuracy(test.end, test.bits)]),
+        Element("td", {}, [fixTarget(formatAccuracy(test.target, test.bits))]),
```

Comment:
Ok, I think what's going on here is you're trying to hide the `target` column, but I think we do that with a class on the table.